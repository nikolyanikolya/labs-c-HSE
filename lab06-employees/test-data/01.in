load test-data/01.data
save all test-data/01.all.out
save developers test-data/01.devs.out
save managers test-data/01.managers.out
save leads test-data/01.leads.out
list-devs
