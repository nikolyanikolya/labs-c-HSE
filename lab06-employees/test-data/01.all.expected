Developer Ivan~Ivanov 100000 ivanovich
Manager Petr~Petrov 80000 HW
LeadDeveloper Sasha~Smith 150000 sasha150 WORLD
Developer John~Ronald~Reuel~Tolkien 100500 jrr
